content/en/aws/managed-streaming-for-kafka/index.md

```diff
@@ -66,7 +68,155 @@ The expected output is something like the following
         },
         "NumberOfBrokerNodes": 3,
         "State": "ACTIVE",
-        "ZookeeperConnectString": "localhost:4515"
+        "ZookeeperConnectString": "localhost:4513"
     }
 }
-{{< / command >}}
\ No newline at end of file
+{{< / command >}}
+
+## Create a kafka topic
+
+In this step of using LocalStack MSK, you create a client machine. You use this client machine to create a topic that produces and consumes data.
+
+First, Install Java on the client machine by running the following command:
+
+{{< command >}}
+sudo yum install java-1.8.0
+{{< / command >}}
+
+Run the following command to download Apache Kafka.
+
+{{< command >}}
+wget https://archive.apache.org/dist/kafka/2.2.1/kafka_2.12-2.2.1.tgz
+{{< / command >}}
+
+Run the following command in the directory where you downloaded the TAR file in the previous step.
+
+{{< command >}}
+tar -xzf kafka_2.12-2.2.1.tgz
+{{< / command >}}
+
+Now, **Go to the kafka_2.12-2.2.1 directory.**
+
+Cluster creation can take a few minutes. To find out whether the cluster you created is ready, run the following command, replacing ```ClusterArn``` with the Amazon Resource Name (ARN) that you obtained above when you created then Cluster.
+
+{{< command >}}
+awslocal kafka describe-cluster --cluster-arn "arn:aws:kafka:us-east-1:000000000000:cluster/EventsCluster"
+{{< / command >}}
+
+Run the following command, replacing ```ZookeeperConnectString``` with the value that you saved after you ran the describe-cluster command.
+
+{{< command >}}
+bin/kafka-topics.sh --create --zookeeper localhost:4513 --replication-factor 1 --partitions 1 --topic LocalMSKTopic
+{{< / command >}}
+
+Your output should be similar to this one
+
+{{< command >}}
+Created topic LocalMSKTopic.
+{{< / command >}}
+
+## Interacting with the topic
+
+In this example we use the JVM truststore to talk to the MSK cluster. To do this, first create a folder named `/tmp` on the client machine. Then, go to the bin folder of the Apache Kafka installation and run the following command, replacing ```java_home``` with the path of your ```java_home```. In this instance, the ```java_home``` is ``` /Library/Internet\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home```.
+
+{{< command >}}
+cp java_home/lib/security/cacerts /tmp/kafka.client.truststore.jks
```

Comment:
Is the truststore creation required? Seems like some of us have been able to communicate with the Kafka cluster with the default settings (although that may be OS-specific). 

Perhaps we should add a short note that this step is optional and may not be required, depending on the operating system environment being used.